bundles/automation/org.eclipse.smarthome.automation.commands/src/main/java/org/eclipse/smarthome/automation/internal/commands/AutomationCommandsPluggable.java

```diff
@@ -43,7 +45,7 @@
  * @author Ana Dimova - Initial Contribution
  * @author Kai Kreuzer - refactored (managed) provider and registry implementation
  */
-@Component(immediate = true)
+@Component(immediate = true, name = "org.eclipse.smarthome.automation.commands")
```

Comment:
Do we require a specific name here?

Reply:
It was like that before in the xml file, which is why I kept it.

Reply:
Just tested it: Works without a specific name.